connectors/opa/lib/catalog_reader.go

```diff
@@ -28,8 +29,11 @@ func (r *CatalogReader) GetDatasetsMetadataFromCatalog(in *taxonomymodels.Policy
 	datasetsMetadata := make(map[string]interface{})
 	datasetID := (in.GetResource()).Name
 	if _, present := datasetsMetadata[datasetID]; !present {
-		objToSend := &pb.CatalogDatasetRequest{CredentialPath: creds, DatasetId: datasetID}
-		info, err := (*r.DataCatalog).GetDatasetInfo(context.Background(), objToSend)
+		// objToSend := &pb.CatalogDatasetRequest{CredentialPath: creds, DatasetId: datasetID}
```

Comment:
Remove commented lines that refer to old code

Reply:
fixed. Thanks.